_usage/migration.md

```diff
@@ -54,8 +55,11 @@ permissions. These are disabled by default in SSMS.
 
 ### Deploying the schema and migrating the data
 
-Make SQL code adjustments as needed. In case you are facing errors or problems,
-we recommend you check out our [discussion about differences in functionality](/docs/usage/limitations-of-babelfish).
+Babelfish implements most T-SQL statements, including `CREATE FUNCTION` and
+`CREATE PROCEDURE`, so it should be easy to port the generated DDL to Babelfish.
+In case you are facing errors or problems, we recommend you check out our
+[discussion about differences in functionality](/docs/usage/limitations-of-babelfish).
+Adjust your DDL code if necessary.
```

Comment:
How about:

Babelfish implements most T-SQL statements, so it should be relatively easy to port the generated DDL to Babelfish. If you are facing errors or problems, we recommend you check out our
[discussion about differences in functionality](/docs/usage/limitations-of-babelfish) and adjust your DDL code if necessary.

Reply:
I kind of wanted to put that in, as a vestige of the removed T-SQL tutorial, since that mostly dealt with functions...
But you are right, it is somehow redundant.

Done in 98282609a66bda5a91db90bda0cfa4a8155ee4cb.